display/private.go

```diff
@@ -161,6 +161,31 @@ func handleHTTP(u string, showInfo bool) {
 	App.Draw()
 }
 
+// handleOther is used by handleURL.
+// It opens links other than Gemini and HTTP and displays Error modals.
+func handleOther(u string) {
+	// The URL should have a scheme due to a previous call to normalizeURL
+	parsed, _ := url.Parse(u)
+	scheme := parsed.Scheme
```

Comment:
This seems unnecessary since the variable is already available, it's not a function or anything.

Reply:
I'm not sure whether I understood correctly. Do you suggest that the function prototype becomes

``` go
func handleOther(u URL)
```

Reply:
No, I'm saying line 169 is unnecessary, as you don't need a `scheme` variable. `parsed.Scheme` can be used everywhere instead. Not a big deal though.

Reply:
Oh, got it! Updated the code, it was a simple fix anyway.